k8s/generate_code.sh

```
@@ -65,16 +65,16 @@
 echo Generating CRD
-controller-gen crd paths=${APIS_DIR}/... output:crd:dir=${CRDS_DIR}
+${GOPATH}/bin/controller-gen crd paths=${APIS_DIR}/... output:crd:dir=${CRDS_DIR}
 pushd ${CRDS_DIR}
-kustomize create --autodetect 2>/dev/null || true
-kustomize edit add label longhorn-manager: 2>/dev/null || true
+${GOPATH}/bin/kustomize create --autodetect 2>/dev/null || true
+${GOPATH}/bin/kustomize edit add label longhorn-manager: 2>/dev/null || true
 if [ -e ${GOPATH}/src/${LH_MANAGER_DIR}/k8s/patches/crd ]; then
   cp -a ${GOPATH}/src/${LH_MANAGER_DIR}/k8s/patches/crd patches
-  find patches -type f | xargs -i sh -c 'kustomize edit add patch --path {}'
+  find patches -type f | xargs -i sh -c '${GOPATH}/bin/kustomize edit add patch --path {}'
```

Shellcheck (0.10.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)